File: public/usage-examples/animations/animation_frame_time-1-example.cpp
#include "splashkit.h"

int main()
{
    animation_script script = load_animation_script("WalkingScript", "kermit.txt");
    animation anim = create_animation(script, "WalkFront");

    write_line("Frame time in current frame:");

    for (int i = 0; i < 5; i++)
    {
        update_animation(anim);
        delay(200);

        write_line("Frame time: " + std::to_string(animation_frame_time(anim)));
    }

    free_animation(anim);
    free_animation_script(script);

    return 0;
}

File: public/usage-examples/animations/animation_frame_time-1-example.py
from splashkit import *

script = load_animation_script("WalkingScript", "kermit.txt")
anim = create_animation(script, "WalkFront")

print("Frame time in current frame:")

for _ in range(5):
    update_animation(anim)
    delay(200)

    print(animation_frame_time(anim))

free_animation(anim)
free_animation_script(script)
